samza-core/src/main/java/org/apache/samza/runtime/RemoteApplicationRunner.java

```diff
@@ -16,17 +16,18 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.samza.system;
+package org.apache.samza.runtime;
 
 import org.apache.samza.operators.StreamGraphBuilder;
 import org.apache.samza.config.Config;
 
+
 /**
- * This class implements the {@link ExecutionEnvironment} that runs the applications in YARN environment
+ * This class implements the {@link ApplicationRunner} that runs the applications in YARN environment
```

Comment:
If this is YARN specific, should probably be called YarnApplicationRunner. If not, should update documentation to say something generic like "runs the application in a remote cluster."